api/soap/mantisconnect.wsdl

```diff
@@ -259,6 +266,41 @@
    <xsd:element name="url" type="xsd:string" minOccurs="0"/>
   </xsd:all>
  </xsd:complexType>
+ <xsd:complexType name="FilterSearchData">
+  <xsd:all>
+   <xsd:element name="project_id" type="tns:IntegerArray" minOccurs="0"/>
+   <xsd:element name="search" type="xsd:string" minOccurs="0"/>
```

Comment:
Do we want a more self-explanatory term than search? E.g. text_search, full_text_search or similar